src/main.ts

```diff
@@ -458,8 +458,14 @@ export interface WAStubMessage {
 export default class WhatsApp {
   private apiSocket: WebSocket;
 
-  private keysPath?: string;
-  private qrPath: string;
+  private parameters : {
+    keysPath: string,
+    qrPath: string,
+    clientName : string,
+    clientShortName : string,
+    restoreSession : boolean
+  }
```

Comment:
```suggestion
  private opts: {
    keysPath: string,
    qrPath: string,
    clientName: string,
    clientShortName: string,
    restoreSession: boolean
  }
```
Adjusting it to my preference and code style.